src/App.css

```diff
@@ -0,0 +1,60 @@
+.app {
+    background-color: white;
+
+} 
+
+.overlay {
+    position: relative;
+}
+
+.Resizer {
```

Comment:
-> lowercase

Reply:
can't.. it doesn't work. That is how the `react-split-pane` docs define it.